app/models/course_data/articles_courses.rb

```diff
@@ -132,7 +131,15 @@ def self.destroy_invalid_records(course, valid_article_ids)
     end
   end
 
-  def self.get_revisions_by_namespaces(namespaces, revisions)
-    revisions.joins(:article).where(articles: { namespace: namespaces })
+  def self.get_revisions_by_namespaces(course)
+    # Select all mainspace article revisions
```

Comment:
@ragesoss, is this fine?

Reply:
See my comment on `#tracked_namespaces`.